test/common/http/utility_fuzz_test.cc

```diff
@@ -41,6 +41,29 @@ DEFINE_PROTO_FUZZER(const test::common::http::UtilityTestCase& input) {
     Http::Utility::extractHostPathFromUri(input.extract_host_path_from_uri(), host, path);
     break;
   }
+  case test::common::http::UtilityTestCase::kPercentEncodingString: {
```

Comment:
Is there any idempotent property you can test here? i.e. decoding the encoded giving you the original

Reply:
I actually don't think there is.